src/main/java/com/bankxpvalue/BankXpValuePlugin.java

```diff
@@ -82,69 +79,45 @@ public void onMenuEntryAdded(MenuEntryAdded event){
             return;
         }
 
-        MenuEntry[] entries = client.getMenuEntries();
-
         if (config.showTutorial()){
-            entries = Arrays.copyOf(entries, entries.length + 1);
-            MenuEntry tutorial = new MenuEntry();
-            tutorial.setOption("Disable tutorial");
-            tutorial.setTarget("");
-            tutorial.setType(MenuAction.WIDGET_FIFTH_OPTION.getId() + 2000);
-            tutorial.setIdentifier(event.getIdentifier());
-            tutorial.setParam0(event.getActionParam0());
-            tutorial.setParam1(event.getActionParam1());
-            entries[entries.length - 1] = tutorial;
-        }
-
-        entries = Arrays.copyOf(entries, entries.length + 1);
-        MenuEntry bankedXp = new MenuEntry();
-        bankedXp.setOption("Toggle Banked XP");
-        bankedXp.setTarget("");
-        bankedXp.setType(MenuAction.WIDGET_FOURTH_OPTION.getId() + 2000);
-        bankedXp.setIdentifier(event.getIdentifier());
-        bankedXp.setParam0(event.getActionParam0());
-        bankedXp.setParam1(event.getActionParam1());
-        entries[entries.length - 1] = bankedXp;
-
-        client.setMenuEntries(entries);
+            client.createMenuEntry(-1)
```

Comment:
Code looks great, but the only thing is is that if you enable the tutorial while you have the overlay position saved outside of the center, the tutorial overlay looks bugged (its position is fixed too).

I'm not sure why anyone would enable the tutorial after already using the plugin, but just to keep everything bug-free, perhaps we should toggle the `keepFixed` option when the user enables the tutorial. We don't even have to keep it enabled, just turning it on and off would reset the position of the overlay to the center and match the tutorial text. What do you think?

<img src="https://user-images.githubusercontent.com/47870467/149426344-6c4b6998-5407-44ea-a520-ccc9acef8ce0.png" width="250">